### PR TITLE
Create the user's schema file

### PR DESCRIPTION
**Pull request by LucasOliveraPolytech**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     "apollo-server-express": "^2.3.1",
     "dotenv": "^6.2.0",
     "express": "^4.16.4",
-    "graphql": "^14.1.1",
+    "graphql": "^14.5.8",
+    "graphql.macro": "^1.4.2",
```
Comment (abelosorio) [inline package.json]: What is this?

File: src/schema/types/user.graphql [hunk 2]
```diff
@@ -0,0 +1,19 @@
+schema {
```
Comment (abelosorio) [inline src/schema/types/user.graphql]: Remove this.

File: src/schema/types/user.graphql [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  type User {
+    id: ID!
+    firstName: String!
+    lastName: String!
+    username: String!
+    password: String!
```
Comment (abelosorio) [inline src/schema/types/user.graphql]: No need to expose password or salt.

File: package.json (hunk 1 above)
Comment (LucasOliveraPolytech) [reply package.json]: I took advice from this site https://create-react-app.dev/docs/loading-graphql-files/. I thought that without this I couldn't handle .graphql files. I didn't end up using it but I forgot to uninstall the package

**Pull request merged**